module/Makefile.in

```diff
@@ -93,7 +93,7 @@ modules_install-Linux: modules_uninstall-Linux-legacy
 		$(if @KERNEL_ARCH@,ARCH=@KERNEL_ARCH@) \
 		KERNELRELEASE=@LINUX_VERSION@
 	@# Remove extraneous build products when packaging
-	if [ -n "$(DESTDIR)" ]; then \
+	if [ -n "$(DESTDIR)" ] && [ "$(DONT_DELETE_MODULES_FILES)" != "1" ]; then \
```

Comment:
I'm fairly sure we added this originally to avoid adding extraneous files to deb and rpm packages.  I'm fine with not changing the behavior for now, but maybe the thing to do is flip this login around and set a flag so we only purge this extra files when building a package.

Reply:
Unless you have strong objections, I'd suggest flipping this logic at a later point. This will allow the fix to get merged now, without causing any unexpected side-effects. The logic flip would have to be communicated with integrators, so it's probably best to do that aligned with a major release.

Reply:
For what it's worth, I test built RPMs with both vanilla `master` and with this PR, and the RPM filelists were the same.